3-java-servlet-web-app/1-Authentication/sign-in-ciam/src/main/java/com/microsoft/azuresamples/msal4j/helpers/GraphHelper.java

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azuresamples.msal4j.helpers;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.Nonnull;
+
+import com.microsoft.graph.authentication.BaseAuthenticationProvider;
+import com.microsoft.graph.core.ClientException;
+import com.microsoft.graph.requests.GraphServiceClient;
+import com.microsoft.graph.requests.GroupCollectionPage;
+import com.microsoft.graph.requests.GroupCollectionRequest;
+import com.microsoft.graph.requests.GroupCollectionRequestBuilder;
+import com.microsoft.graph.models.Group;
+
+/**
+ * GraphHelper class handles creating a Graph SDK client (IGraphServiceClient)
+ * and has functions for common Microsoft Graph calls
+ */
+public class GraphHelper {
```

Comment:
This sample doesn't call Graph. Why is this here?

Reply:
I originally copied most of the code from the basic "sign-in" sample, and if you look [here](https://github.com/Azure-Samples/ms-identity-msal-java-samples/blob/0f338b0d20fd8d1e540bd24b1fee2f300643a0b9/3-java-servlet-web-app/1-Authentication/sign-in/src/main/java/com/microsoft/azuresamples/msal4j/helpers/AuthHelper.java#L92) you can see how it could be used: apparently just as a way to handle a user in too many groups.

It might be helpful for the samples that deal with groups but isn't relevant here, so in the latest commit I just deleted this class and it's potential references.